notebooks/Project and Developer Similarity.ipynb

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
<p>How do you think, would that be useful to explaining a bit of the dataframe magic, like <code>topics_identifiers_df.iloc[:, 2:].values.T</code>? </p><p> </p><p>IDK, but this may not be something everyone is immediately familiar with, but can be useful takeaway in a long run.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mloncode/devfest2019-workshop/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Thanks for noticing the obscure code, I will refactor a bit (simplify the indexes to avoid manipulating columns) and also add comments.</p>